CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.23.0-rc.0 / 2020-11-20
+
+* [CHANGE] Make the React UI default. #8142
+* [CHANGE] The following metrics were removed/renamed in remote write. #6815
+     - `succeeded_samples_total` was removed and `samples_total` was introduced for all the samples attempted to send.
+     - `sent_bytes_total` was removed and was replaced with `samples_bytes_total` and `metadata_bytes_total`.
+     - `failed_samples_total` -> `samples_failed_total` .
+     - `retried_samples_total` -> `samples_retried_total`.
+     - `dropped_samples_total` -> `samples_dropped_total`.
+     - `pending_samples` -> `samples_pending`.
+* [FEATURE] Allow metric metadata to be propagated via remote write. The following new metrics were introduced: `metadata_total` (total number of metadata sent to remote storage), `metadata_failed_total` (total number of metadata send failed, non recoverable errors), `metadata_retried_total` (total number of metadata send retried after facing recoverable errors), `metadata_bytes_total` (total bytes send for metadata after compression). #6815
```

Comment:
Only the metric names need to be mentioned, this isn't the place for metric docs.